### PR TITLE
Fixes multiline markdown link text

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/eventsource/withcredentials/index.md [hunk 1]
```diff
@@ -29,8 +29,7 @@ var evtSource = new EventSource('sse.php');
 console.log(evtSource.withCredentials);
 ```
 
-> **Note:** You can find a full example on GitHub — see [Simple SSE
-> demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
+> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP.](https://github.com/mdn/dom-examples/tree/master/server-sent-events)
```
Comment (teoli2003) [inline files/en-us/web/api/eventsource/withcredentials/index.md:32]:
```suggestion
> **Note:** You can find a full example on GitHub — see [Simple SSE demo using PHP](https://github.com/mdn/dom-examples/tree/master/server-sent-events).
```

**Review by teoli2003 — APPROVED** [review]
One nit and this is good to go.

**Comment by OnkarRuikar** [comment]
There are a lot of `[...PHP.]` cases lets do them in one go in a separate PR.

**Review by teoli2003 — APPROVED** [review]
Yep!

**Pull request merged**